docs/FAQ.md

```diff
@@ -28,6 +28,7 @@
     - [How to enable and get the debug logs of docker daemon?](#how-to-enable-and-get-the-debug-logs-of-docker-daemon)
     - [New user permissions may not be effective immediately after updating](#new-user-permissions-may-not-be-effective-immediately-after-updating)
     - [Authentication information is not given in the correct format on direct REST API calls](#authentication-information-is-not-given-in-the-correct-format-on-direct-rest-api-calls)
+    - [Portal doesn't list all repositories/tags](#Portal-doesn't-list-all-repositories/tags)
```

Comment:
Recommend rephrasing this to "Why does the Azure Portal not list all my repositories?"

Reply:
Rephrased.